deploy/stacks/cognito.py

```diff
@@ -26,6 +28,7 @@ def __init__(
         vpc=None,
         prod_sizing=False,
         internet_facing=True,
+        custom_waf_rules=None,
```

Comment:
I have been tracking this parameter upstream and I cannot see it neither in the `cdk.json` nor in the backend stack. Should we re-enable it and add it to the cdk.json template and docs? In such case, maybe it is worth opening a separate PR. wdyt?